src/Components/ArticlePage.css

```diff
@@ -0,0 +1,56 @@
+.article-page-main__container {
+    height: 700px;
+    width: 100%;
+    background-color: gray;
+    flex-direction: row;
+
+    border: green 2px solid;
+
+    transition: transform 0.2s ease-in-out;
+}
+
+.article-page-main__container:hover {
+  transform: scale(1.1);
+}
+
+.article-page-image__container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    height: 600px;
```

Comment:
A fixed unit on images can be problematic - consider a relative one - em for example